src/unpackertool/plugins/upx/upx_stub.h

```diff
@@ -11,9 +11,11 @@
 
 #include "retdec/loader/loader.h"
 #include "retdec/unpacker/plugin.h"
-#include "retdec/unpacker/dynamic_buffer.h"
+#include "retdec/utils/dynamic_buffer.h"
 #include "retdec/unpacker/unpacking_stub.h"
 
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.